test/fileqcTest.js

```diff
@@ -202,11 +206,46 @@ describe('FileQcController', () => {
   revertFprDb();
 });
 
+describe('available constants', () => {
+  beforeEach(async () => {
+    await cmd.run(
+      'sqlite3 ' +
+        process.env.SQLITE_LOCATION +
+        '/fpr.db < ' +
+        test_fpr_creation
+    );
+    await cmd.run(
```

Comment:
Why is this duplicated?

Reply:
It isn't quite duplicated -- the first one is the FPR creation file, the second one is the migration file.

Reply:
Oh. I see now. Sorry.

Reply:
No worries - the formatting made that less obvious!